lib/api/cln/cln_client.dart

```diff
@@ -55,7 +55,16 @@ class CLNApi extends AppApi {
             jsonResponse as Map<String, dynamic>,
             snackCase: !mode.withCamelCase()));
     var listFunds = await this.listFunds();
+    var invoicesList = await listInvoices(status: "paid");
```

Comment:
In addition, consider tha the listinvoices can be very large and this mean that this call on the internet take years to finish

We already scan this list for the payments, why not trying to scan a single time this list?